.github/workflows/publish.yml

```diff
@@ -0,0 +1,47 @@
+
+name: Publish to PyPI
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  packages:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
```

Comment:
It is probably safe to do a `'3.x'` here. It would ensure that any Python 3 available in the image could be used.